modules/likes.php

```diff
@@ -1020,18 +994,10 @@ function get_options() {
 	 * similar logic and filters apply here, too.
 	 */
 	function is_likes_visible() {
-
 		global $post, $wp_current_filter;              // Used to apply 'sharing_show' filter
-		// @todo: Remove this block when 4.5 is the minimum
```

Comment:
We can't merge this yet. 4.5 won't be the minimum version for Jetpack until the release after 4.6 ships.